crates/cheatcodes/src/lib.rs

```diff
@@ -45,6 +45,7 @@ mod json;
 
 mod script;
 pub use script::{ScriptWallets, ScriptWalletsInner};
+use spec::Status;
```

Comment:
move to L17

Reply:
moved in https://github.com/foundry-rs/foundry/pull/8883/commits/877d6fb4ca78d63fa53aabe955847102eb2b7e04